src/components/Tabs/ScatterPlotTab.tsx

```diff
@@ -496,6 +496,8 @@ export default memo(function ScatterPlotTab(props: ScatterPlotTabProps): ReactEl
     // Sort data into buckets
     for (let i = 0; i < xData.length; i++) {
       const objectId = objectIds[i];
+      const isMinMaxNaN = Number.isNaN(colorMaxValue) && Number.isNaN(colorMinValue);
```

Comment:
Caused by a change I mentioned where we're no longer using `Infinity` as our placeholder value.